src/main/java/org/example/service/ConnectionService.java

```diff
@@ -0,0 +1,55 @@
+package org.example.service;
+
+import java.sql.*;
+
+
+public class ConnectionService {
+    private static final String DATABASE_NAME = "postgres";
+    private static final String DATABASE_USER = "postgres";
+    private static final String DATABASE_PASSWORD = "postgres";
+    private static final String DRIVER_POSTGRESQL = "org.postgresql.Driver";
+    private static final String SCHEMA = "jdbc:postgresql";
+    private static final String HOST = "developers-db";
+    private static final String PORT = "5432";
+    private final Connection connection;
+    private static ConnectionService INSTANCE;
+
+    public ConnectionService() {
+        connection = connectToDb();
+    }
+
+    public Connection getConnection() {
+        return connection;
+    }
+
+    public Connection connectToDb() {
+        Connection connectionLocal;
+        try {
+            Class.forName(DRIVER_POSTGRESQL);
+            connectionLocal = DriverManager.getConnection(getDatabaseUrl(), DATABASE_USER, DATABASE_PASSWORD);
+        } catch (Exception e) {
+            System.err.println("Error: check connection methods!");
+            e.printStackTrace();
+            throw new RuntimeException(e);
+        }
+
+        if (connectionLocal == null) {
```

Comment:
This IF statement is redundant.
Remove it.

Reply:
fixed